packages/ng/statusBadge/statusBadge.component.html

```diff
@@ -0,0 +1 @@
+{{label}}
```

Comment:
Can we have something else that just a string as content? Maybe we should use `ng-content` to handle that in a more intuitive fashion?

Also, one-liner templates are better as inline `template` rather than in their own file.